src/ucp/proto/proto_select.c

```diff
@@ -137,9 +137,9 @@ static void ucp_proto_select_perf_str(const ucs_linear_func_t *perf,
                                       char *time_str, size_t time_str_max,
                                       char *bw_str, size_t bw_str_max)
 {
-    /* Estimated time */
-    snprintf(time_str, time_str_max, "%.0f + %.3f * N", perf->c * 1e9,
-             perf->m * 1e9);
+    /* Estimated time per 1024 bytes*/
+    snprintf(time_str, time_str_max, "%.2f + %.3f * N", perf->c * 1e9,
```

Comment:
maybe define FMT and ARG macro for this
`#define UCP_PROTO_PERF_FUNC_SHORT_FMT  "%.2f ns + %.3f ns/kB"`
can reuse it in multiple places